packages/sdk/src/evm/core/sdk.ts

```diff
@@ -262,11 +262,12 @@ export class ThirdwebSDK extends RPCConnectionHandler {
     options: SDKOptions = {},
     storage?: IThirdwebStorage,
   ) {
-    const apiKeyType = typeof window !== "undefined" ? "clientId" : "secretKey";
+    const apiKeyType = typeof window !== "undefined" ? "clientId" : options.authToken ? "authToken" : "secretKey";
```

Comment:
I don't think we want to check `authToken` here. the option to pass in the `authToken` is probably more internal to our own usage of SDKs than to external usage of the SDK